simbad/rotsearch/amore_search.py

```diff
@@ -329,4 +338,61 @@ def submit_chunk(chunk_scripts, run_dir, nproc, job_name, submit_qtype, submit_q
                  max_array_jobs=nproc, queue=submit_queue, permit_nonzero=True)
         interval = int(math.log(len(chunk_scripts)) / 3)
         interval_in_seconds = interval if interval >= 5 else 5
-        j.wait(interval=interval_in_seconds, monitor=monitor)
+        j.wait(interval=interval_in_seconds, monitor=monitor, check_success=self.rot_succeeded_log)
+
+    @staticmethod
+    def _rot_job_succeeded(amore_z_score):
+        """Check values for job success"""
+        return amore_z_score > 10
+
+    @staticmethod
+    def _mr_job_succeeded(r_fact, r_free):
+        """Check values for job success"""
+        return r_fact < 0.45 and r_free < 0.45
+
+    def rot_succeeded_log(self, log):
+        """Check a rotation search job for it's success
+
+        Parameters
+        ----------
+        log : str
+           The path to a log file
+
+        Returns
+        -------
+        bool
+           Success status of the rot run
+
+        """
+        rot_prog, pdb = os.path.basename(log).replace('.log', '').split('_', 1)
+        rp = simbad.parsers.rotsearch_parser.RotsearchParser(log)
+        dat_model = [s for s in self.simbad_dat_files if pdb in s][0]
+        score = simbad.rotsearch.amore_score.AmoreRotationScore(
+            pdb, dat_model, rp.alpha, rp.beta, rp.gamma,
+            rp.cc_f, rp.rf_f, rp.cc_i, rp.cc_p, rp.icp,
+            rp.cc_f_z_score, rp.cc_p_z_score, rp.num_of_rot
+        )
+        results = [score]
+        if self._rot_job_succeeded(rp.cc_f_z_score) and pdb not in self.tested:
+            self.tested.append(pdb)
+            output_dir = os.path.join(self.work_dir, "mr_search")
+            mr = simbad.mr.MrSubmit(mtz=self.mtz,
+                                    mr_program=self.mr_program,
+                                    refine_program='refmac5',
+                                    refine_type=None,
+                                    refine_cycles=0,
+                                    output_dir=output_dir,
+                                    tmp_dir=self.tmp_dir,
+                                    timeout=30)
+            mr.mute = True
+            mr.submit_jobs(results,
+                           nproc=1,
+                           process_all=True,
+                           submit_qtype='local',
```

Comment:
@fsimkovic I've set submit_qtype as 'local' here. I've not tested this on the SGE cluster, but my logic was that it would prevent the job from being queued behind the existing rotation jobs. It's just occurred to me however that it might be better to set submit_qtype to SGE with an additional priority argument. What do you think?